go/list/list_test.go

```diff
@@ -0,0 +1,118 @@
+package list
+
+import (
+	"testing"
+)
+
+// Can initialize an empty list
+func TestInitEmptyList(t *testing.T) {
+    l := New[int]()
```

Comment:
We usually use tabs instead of spaces in the codebase, to avoid future conflict/changes to this file, it would be best to convert all the spaces to tabs

Reply:
Could you please provide more detail on where should I convert spaces to tabs so that I can make the necessary adjustments accordingly

Reply:
You can/should use `gofmt` on the file and that should enforce tabs everywhere. For example:
```
❯ cat /tmp/foo.go
package main

func main() {
  if foo {
    fmt.Println("hi")
  }
}
❯ gofmt /tmp/foo.go
package main

func main() {
        if foo {
                fmt.Println("hi")
        }
}

❯ gofmt -w /tmp/foo.go

❯ cat /tmp/foo.go
package main

func main() {
        if foo {
                fmt.Println("hi")
        }
}
```

Reply:
Thanks! 
I have made the changes please have a look.

Reply:
Hey @ChaitanyaD48, there are still a bunch of spaces instead of tabs everywhere in the file. It's nit-picking, but I think converting everything to tabs will avoid future unnecessary changes to this file.

Reply:
@ChaitanyaD48 You should make sure that your editor is set up to do this automatically for you. If you for example use Visual Studio Code with the Go plugin, it happens automatically.

Reply:
You also see this in the failing `Static Code Checks` job which enforces the style.

Reply:
Got it! I made the changes and I guess now the workflow should not fail